es6-shim.js

```diff
@@ -205,7 +205,15 @@
     return result;
   };
 
+  var bindContext = function (func, context) {
+    return function () {
+      return func.apply(context, arguments);
+    };
+  };
+
   var ES = {
+    Call: bindContext(Function.prototype.call, Function.prototype.apply),
```

Comment:
I've released https://www.npmjs.com/package/es-abstract for the `es7-shim`, which I eventually intend to add to the `es6-shim` once I get a build process going. For now, please consider any ES function inside `es-abstract` fair game for copy-paste into the `es6-shim`, including `ES.Call` :-) Thanks for this thread, it helped lead me to the idea.